specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2021-09-01-privatepreview/examples/Migrations_Create.json

```diff
@@ -0,0 +1,152 @@
+{
+  "parameters": {
+    "api-version": "2021-09-01-privatepreview",
+    "targetDBServerSubscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "targetDBServerResourceGroupName": "testrg",
+    "targetDBServerName": "testtarget",
+    "migrationName": "testmigration",
+    "parameters": {
+      "properties": {
+        "sourceDBServerResourceId": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/testrg/providers/Microsoft.DBForPostgreSql/servers/testsource",
+        "secretParameters": {
+          "adminCredentials": {
+            "sourceServerPassword": "testsourcepassword",
+            "targetServerPassword": "testtargetpassword"
+          },
+          "aadApp": {
+            "aadSecret": "testaadsecret",
+            "clientId": "cccccccc-cccc-cccc-cccc-cccccccccccc",
+            "tenantId": "tttttttt-tttt-tttt-tttt-tttttttttttt"
+          }
+        },
+        "dBsToMigrate": [
+          "db1",
+          "db2",
+          "db3",
+          "db4"
+        ],
+        "migrationResourceGroup": {
+          "resourceId": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/testrg",
+          "subnetResourceId": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/testrg/providers/Microsoft.Network/virtualNetworks/testvnet/subnets/default"
+        }
+      },
+      "location": "westus"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "migrationId": "mmmmmmmm-mmmm-mmmm-mmmm-mmmmmmmmmmmm",
+          "migrationName": "testmigration",
+          "migrationDetailsLevel": "Default",
+          "currentStatus": {
+            "state": "InProgress",
+            "error": "",
+            "currentSubStateDetails": {
+              "currentSubState": "PerformingPreRequisiteSteps"
+            }
+          },
+          "sourceDBServerMetadata": {
+            "location": "westus",
+            "version": "10",
+            "storageMB": 1024,
+            "sku": {
+              "name": "B_Gen5_2",
+              "tier": "Basic"
+            }
+          },
+          "targetDBServerMetadata": {
+            "location": "westus",
+            "version": "11",
+            "storageMB": 2048,
+            "sku": {
+              "tier": "GeneralPurpose",
+              "name": "Standard_D4s_v3"
+            }
+          },
+          "sourceDBServerResourceId": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/testrg/providers/Microsoft.DBForPostgreSql/servers/testsource",
+          "secretParameters": null,
```

Comment:
I see your comment is resolved, but I don't see it resolved. Can you remove "secretParameters": null, in all the responses.